.ci/deps.sh

```diff
@@ -16,4 +15,4 @@ for dep_version in "${dep_versions[@]}" ; do
 
   pip3 install -q -r test-requirements.txt
   pip3 install -q -r requirements.txt
-done
\ No newline at end of file
+done
```

Comment:
In emacs it always adds a newline at the end of the file and always clears up trailing whitespaces. Is adding a newline at the end of the file is bad ?